### ISSUE TITLE
test_keyboard.py fails

### ISSUE THREAD

**Issue by lordmauve** [lordmauve/pgzero#25]
Originally reported by: **Anonymous**

----------------------------------------

When running tests, test_keyboard.py fails in the assert_warning context manager.

```
#!

/usr/local/Cellar/python3/3.4.3_2/Frameworks/Python.framework/Versions/3.4/lib/python3.4/contextlib.py:66: in __exit__
    next(self.gen)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    @contextmanager
    def assert_warning():
        with warnings.catch_warnings(record=True) as w:
            # warnings.simplefilter("always")
            yield
>       assert len(w) > 0, \
            "Expected a warning, but no warnings received."
E       AssertionError: Expected a warning, but no warnings received.
E       assert 0 > 0
E        +  where 0 = len([])

test/test_keyboard.py:16: AssertionError
```

----------------------------------------
- Bitbucket: https://bitbucket.org/lordmauve/pgzero/issue/25

**Comment by lordmauve** [comment]
*Original comment by* **Xavier Oliver (Bitbucket: [xoliver](https://bitbucket.org/xoliver), GitHub: [xoliver](https://github.com/xoliver))**:

----------------------------------------

Fix PR coming

**Comment by lordmauve** [comment]
*Original comment by* **Daniel Pope (Bitbucket: [lordmauve](https://bitbucket.org/lordmauve), GitHub: [lordmauve](https://github.com/lordmauve))**:

----------------------------------------

Fixed by fc27e6d
